todo_app/data/view_model.py

```diff
@@ -0,0 +1,35 @@
+from todo_app.data.item import Item
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+ 
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self):
+        todo_items = []
+        for item in self._items:
+            if item.status == "To Do":
+                todo_items.append(item)
+        return todo_items
```

Comment:
Not necessarily a recommendation, but I always think interesting to see a comparison with a list comprehension approach:
```python
    @property
    def todo_items(self):
        return [item for item in self._items if item.status == "To Do"]
```